rockspec/apisix-3.0-0.rockspec

```diff
@@ -0,0 +1,103 @@
+--
```

Comment:
The rockspec should contain patch version, see https://github.com/apache/apisix/blob/master/rockspec/apisix-2.15.0-0.rockspec

Reply:
done